Geometry_Engine/Query/IsPolylinear.cs

```diff
@@ -46,27 +51,51 @@ public static bool IsPolylinear(this PolyCurve curve)
 
         /***************************************************/
 
+        [Description("Returns if the curve is polylinear or not. For a Arc this always returns false.")]
+        [Input("curve", "The Arc to check if it is polylinear.")]
+        [Output("isLinear", "Return true if the curve is polylinear. For a Arc this always return false.")]
         public static bool IsPolylinear(this Arc curve)
         {
             return false;
         }
 
         /***************************************************/
 
+        [Description("Returns if the curve is polylinear or not. For a Circle this always returns false.")]
+        [Input("curve", "The Circle to check if it is polylinear.")]
+        [Output("isLinear", "Return true if the curve is polylinear. For a Circle this always return false.")]
         public static bool IsPolylinear(this Circle curve)
         {
             return false;
         }
 
         /***************************************************/
 
+        [Description("Returns if the curve is polylinear or not. For a Ellipse this always returns false.")]
+        [Input("curve", "The Ellipse to check if it is polylinear.")]
+        [Output("isLinear", "Return true if the curve is polylinear. For a Ellipse this always return false.")]
+        public static bool IsPolylinear(this Ellipse curve)
+        {
+            //Could argue that the ellipse is polylinear if one of the radii is 0, but given the use of this method,
```

Comment:
This is a Pandora's box - same could be said about linearity of an infinitely large arc (or even a circle, why not). So supportive towards this approach